src/forwarder.ts

```diff
@@ -140,3 +151,80 @@ export async function describeSubscriptionFilters(aws: Aws, logGroupName: string
     return [];
   }
 }
+
+// Helper functions to validate we have a particular log group and if we should subscribe to it
+function validateApiGatewaySubscription(resource: any, subscribe: boolean) {
+  return resource.Properties.LogGroupName.startsWith("/aws/api-gateway/") && subscribe;
+}
+function validateHttpApiSubscription(resource: any, subscribe: boolean) {
+  return resource.Properties.LogGroupName.startsWith("/aws/http-api/") && subscribe;
+}
+function validateWebsocketSubscription(resource: any, subscribe: boolean) {
+  return resource.Properties.LogGroupName.startsWith("/aws/websocket/") && subscribe;
+}
+
+function shouldNotSubscribe(resource: any, forwarderConfigs: ForwarderConfigs) {
```

Comment:
nit: By convention, we usually use positives for these kinds of functions, eg 'shouldSubscribe'. This is so we avoid using double negatives in our logic, eg 
```typescript
!shouldNotSubscribe() // <-- this is a double negative
shouldNotSubscribe()
// vs
shouldSubscribe()
!shouldSubscribe()
```